src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_azcopy_scenarios.py

```diff
@@ -72,3 +74,119 @@ def test_storage_blob_azcopy_sync(self, resource_group, storage_account_info, te
             test_dir, container, storage_account))
         self.cmd('storage blob list -c {} --account-name {}'.format(
             container, storage_account), checks=JMESPathCheck('length(@)', 0))
+
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer()
+    @StorageTestFilesPreparer()
+    def test_storage_blob_azcopy_remove(self, resource_group, storage_account_info, test_dir):
+        storage_account, _ = storage_account_info
+        container = self.create_container(storage_account_info)
+
+        # sync directory
+        self.cmd('storage blob sync -s "{}" -c {} --account-name {}'.format(
+            test_dir, container, storage_account))
+        self.cmd('storage blob list -c {} --account-name {}'.format(
+            container, storage_account), checks=JMESPathCheck('length(@)', 41))
+
+        self.cmd('storage remove -c {} -n readme --account-name {}'.format(
+            container, storage_account))
```

Comment:
It looks that you do not provide a check for this command. In this way, how do you know test results?

Reply:
The test checked the list of files after remove command. The total files became 40 from 41 after removing one file. See the next line for the check.